core/src/main/java/com/sequenceiq/cloudbreak/reactor/CollectDownscaleCandidatesHandler.java

```diff
@@ -160,17 +155,16 @@ private void fillDiscoveryFQDNInVolumeSetIfEmpty(List<InstanceMetaData> removabl
         }
     }
 
-    private Set<Long> collectCandidates(CollectDownscaleCandidatesRequest request, Stack stack, int defaultRootVolumeSize)
+    private Set<Long> collectCandidates(CollectDownscaleCandidatesRequest request, Stack stack)
             throws CloudbreakException {
-        LOGGER.debug("Collecting candidates for downscale based on " + CollectDownscaleCandidatesRequest.class.getSimpleName() + " [{}] and "
-                + Stack.class.getSimpleName() + " [{}] and default root volume size [{}]", request.toString(), "stackCrn: " + stack.getResourceCrn(),
-                defaultRootVolumeSize);
+        LOGGER.debug("Collecting candidates for downscale based on CollectDownscaleCandidatesRequest request [{}] and "
```

Comment:
This will log `Collecting candidates for downscale based on CollectDownscaleCandidatesRequest request [CollectDownscaleCandidatesRequest{...}] ...`
Maybe a bit redundant.

Reply:
yes, makes sensen